README.md

```diff
@@ -84,7 +84,7 @@ self.client = [PTPusher pusehrWithKey:@"YOUR_API_KEY" delegate:self encrypt:YES
 
 ### Subscribe to channels
 
-Channels are a way of filtering the events you want your application to receive. In addition, private and presence channels can be used to control access to events and in the case of presence channels, see who else is subscribing to events. For more information on channels, see the Pusher documentation.
+Channels are a way of filtering the events you want your application to receive. In addition, private and presence channels can be used to control access to events and in the case of presence channels, see who else is subscribing to events. For more information on channels, see the [Pusher Channels documentation](https://pusher.com/docs/channels).
```

Comment:
Plus, the docs should link to the docs on using Channels - ie private/presence/public. Not the generic Channels docs.